static/js/tools/choropleth/choropleth.tsx

```diff
@@ -21,25 +21,40 @@
 import React, { Component } from "react";
 import axios from "axios";
 import * as d3 from "d3";
+import ReactDOM from 'react-dom';
+
+type PropsType = {}
+
+// TODO: get rid of "any" type.
```

Comment:
```suggestion
// TODO(eduardo): get rid of "any" type.
```

Or whatever your GitHub name is.